### ISSUE TITLE
media player did not show up for item "Magic Eye"

### ISSUE THREAD

**Issue by htpvu** [jhu-idc/iDC-general#464]
<img width="1398" alt="Screen Shot 2022-01-06 at 1 36 12 PM" src="https://user-images.githubusercontent.com/11050142/148434940-dfd5da6c-e7f1-492d-89ae-2e519e1b4886.png">

**Comment by bseeger** [comment]
I just fixed this. It was because the media was not marked as Original File and Service File.  The system looks for the Media tagged with Service File to give to the media player - so in this case it had nothing to grab, hence nothing showing up in the viewer. As soon as I added Service File, it worked as expected. 

What I did: 
- Went to the item's "Media" tab. 
- Clicked on the Media's "Edit". 
- On that page I set Media Use to "Original File" and "Service File", 
- saved and went back to view the object. 

Note:  Original File is useful but not the key here - setting it to Service File is what does the trick. This item must have been ingested w/o that data set on the media?  I'm not sure.
